src/config.rs

```diff
@@ -362,6 +364,16 @@ impl Cfg {
         Toolchain::from(self, name)
     }
 
+    pub fn get_toolchains_from_glob(
+        &self,
+        pattern: Pattern,
+    ) -> Result<impl Iterator<Item = Toolchain<'_>>> {
```

Comment:
I'm calling the function `...glob` since "pattern" could mean `std::pattern::Pattern`, which is not the meaning here.